scripts/insert_into_appointments.sql

```diff
@@ -1 +1,30 @@
 -- insert appointments creating connections between doctors and patients
+INSERT INTO appointments(start_at, patient_id, doctor_id)
```

Comment:
This would have worked if you had created your tables correctly.  However, even if that were the case, it doesn't fulfill exactly what was asked of you.